sdk/cognitivelanguage/ai-language-text/test/public/analyzeBatch.spec.ts

```diff
@@ -303,6 +335,34 @@ matrix([FIXME3] as const, async (authMethod: AuthMethod) => {
             });
           });
 
+          it("healthcare with known documents type", async function () {
+            const docs = [
+              "The patient is a 54-year-old gentleman with a history of progressive angina over the past several months.",
+              "Prescribed 100mg ibuprofen, taken twice daily.",
+              "Patient does not suffer from high blood pressure.",
+            ];
+            const poller = await client.beginAnalyzeBatch(
+              [
+                {
+                  kind: AnalyzeBatchActionNames.Healthcare,
+                  fhirVersion: KnownFhirVersion["4.0.1"],
+                  documentType: KnownDocumentType.DischargeSummary,
+                },
+              ],
+              docs,
+              "en",
+              {
+                updateIntervalInMs: pollingInterval,
+              }
+            );
+            await poller.pollUntilDone();
+
+            const results = await poller.pollUntilDone();
+            await assertActionResults(results, expectation32, {
+              excludedAdditionalProps: ["reference", "id", "fullUrl", "value", "date", "period"],
```

Comment:
Nitpick: these shows up multiple times now, perhaps we should have that list in a variable.